functions/__fzf_grep_current_dir.fish

```diff
@@ -0,0 +1,36 @@
+function __fzf_grep_current_dir --description "Search current directory content for $token. Replace the current token with the selected file paths."
+    # Make sure that fzf uses fish to execute __fzf_preview_file.
+    # See similar comment in __fzf_search_shell_variables.fish.
+    set --local --export SHELL (command --search fish)
+
+    set token (commandline --current-token | string unescape)
+
+    set fzf_arguments --multi --ansi --exact --phony --reverse --query=$token \
+        --preview='rg --ignore-case --pretty --context 10 {q} {}' \
+        --bind "change:reload:rg --ignore-case --files-with-matches --no-ignore-vcs {q} || true"
```

Comment:
why `--no-ignore-vcs`?

Reply:
Yeah this should be removed.

I believe it would be best to allow user to configure some default argument to ripgrep